app/controllers/users.js

```diff
@@ -0,0 +1,19 @@
+const logger = require('../logger'),
+  users = require('../services/users');
+
+exports.userRegistration = (req, res, next) => {
+  logger.info('POST method start. User registration.');
+  if (req.query) {
+    const user = req.query;
```

Comment:
The best way to handle multiple attributes in an object, is to send it inside the body section. Extract the info from `body` instead of `query`.